### PR TITLE
Update fast tree argument help text

### PR DESCRIPTION
**Pull request by jwood803**
Proposed update for issue #223.

### CODE REVIEW

**Comment by jwood803** [comment]
It seems that `TestEntryPoints.EntryPointCatalog` is also failing with this change. Unfortunately, I haven't figured out why yet, but I'm still looking into it. :)

**Comment by jwood803** [comment]
And found it! Had to update the `core_manifest.json` file with the updated help text.

**Comment by TomFinley** [comment]
Indeed. Our API is autogenerated from classes in the runtime. The test `EntryPointCatalog` generates an alternate file and compares against it (I use beyond compare myself, but there are free alternatives I think like .) Then there's this separate `RegenerateCSharpApi` test to actually change the "public" API itself. It's a little involved.

File: src/Microsoft.ML/CSharpApi.cs [hunk 1]
```diff
@@ -3966,7 +3966,7 @@ public sealed partial class FastForestBinaryClassifier : Microsoft.ML.Runtime.En
             public int MinDocumentsInLeafs { get; set; } = 10;
 
             /// <summary>
-            /// Number of weak hypotheses in the ensemble
+            /// Total number of decision trees to create in the ensemble
```
[the remark this answers is not in the document]
Comment (jwood803) [reply src/Microsoft.ML/CSharpApi.cs]: Actualy, @mairaw, in `FastTreeArguments.cs` the other help text properties don't have those in them. Is it still ok to add them to this one or should I do it to all of them to be consistent in the file?

**Pull request merged**